docs/source/_static/docversions.js

```diff
@@ -0,0 +1,45 @@
+var versions = ['master',  '1.7.0', '1.6.0',
+                '1.5.1', '1.5.0', '1.4.0', '1.3.1', '1.3.0', '1.2.0',
+                '1.1.0', '1.0.1', '1.0.0', '0.4.1', '0.4.0', '0.3.1',
+                '0.3.0', '0.2.0', '0.1.12'];
+
+function insert_version_links() {
+    for (i = 0; i < versions.length; i++){
+        open_list = '<li>'
+        label = 'v' + versions[i];
+        switch (label){
+            case 'vmaster':
+                label = 'master (unstable)';
+                break;
+            case '1.7.0':
+                label += ' (rc1)';
+                break;
+            case '1.6.0':
+                label += ' (stable release)';
+                break;
+        }
+        if (typeof(DOCUMENTATION_OPTIONS) !== 'undefined') {
+            if (DOCUMENTATION_OPTIONS['VERSION'] == versions[i])
+            {
+                open_list = '<li id="current">'
+            }
+        }
+        const pathPattern = /docs\/(build\/html|stable|master|[0-9.rc]+)(.*)/;
```

Comment:
whether you build locally or deploy to pytorch/pytorch.github.io, the document URL will look like
```
file:///home/user/pytorch/docs/build/html/generated/torch.nn.parameter.Parameter.html
or
https://pytorch.org/docs/1.6.0/generated/torch.nn.parameter.Parameter.html
```
So the `docs` part is a consistent anchor, then either `build/html` or one of the versions (`stable', `master`, `1.6.0` etc), then the actual document name we are interested in.